exo-go/test_helpers/shared_feature_context.go

```diff
@@ -56,17 +57,20 @@ func SharedFeatureContext(s *godog.Suite) {
 
 	s.Step(`^running "([^"]*)" in the terminal$`, func(command string) error {
 		var err error
-		childOutput, err = run(command)
+		childOutput, err = processHelpers.Run(command, cwd)
```

Comment:
Parse the string into shell words here to feed it as an array into this command.